src/web/components/ClientSideTokenGeneration/CstgAddDomainDialog.spec.tsx

```diff
@@ -95,9 +108,13 @@ describe('CstgDomainAddDomainDialog', () => {
       />
     );
 
+    await user.type(screen.getByRole('textbox', { name: 'newDomains' }), 'newDomains');
```

Comment:
nit: perhaps we could make the input and test case clearer here by doing something like:
```
await user.type(screen.getByRole('textbox', { name: 'newDomains' }), 'invalid domain');
```